canonical-web-and-design/charmhub.io
# Charmhub libraries page - layout improvements

## Done
- Ensure the layout uses the available screen space optimally at all screen sizes
- Small refactor to reduce the heavy use of class names on many elements, as a first step towards making it a modular component based page. 

## How to QA
- Visit the demo in your browser, link commented below
- Double check the flask conditionals, ensure the logic is sound
- Resize the screen, check at various resolutions
 
## Issue / Card
Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3637 (on the vanilla repo)

## Screenshots
![image](https://user-images.githubusercontent.com/2741678/111803668-28d74c80-88c7-11eb-87c7-d45ec8f3bd7d.png)
